packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -325,10 +330,10 @@ const Tab = React.forwardRef((props, forwardedRef) => {
       role='tab'
       tabIndex={isActive ? 0 : -1}
       aria-selected={isActive}
-      aria-controls={`${idPrefix}-panel-${value}`}
+      aria-controls={`${idPrefix}-panel-${value.replaceAll(' ', '-')}`}
       ref={useMergedRefs(tabRef, forwardedRef, setInitialActiveRef)}
       {...rest}
-      id={`${idPrefix}-tab-${value}`}
+      id={`${idPrefix}-tab-${value.replaceAll(' ', '-')}`}
```

Comment:
Was this change necessary? Or was it addressing something else?

Also, might be a contrived example, but what if users have two tab items with values "Item 1" and "Item-1". Will there be two `iui-tab`s with the same id?

Reply:
It's unrelated. Spaces are not valid in `id`, and `aria-controls` can accept multiple space-separated ids. I noticed this in our stories when debugging the main issue.

I can't think of a better way to address this, so I think it's fine. We do the same thing in ComboBox iirc